### PR TITLE
[ID] Update `Game modifier/Hard Rock`

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 1]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+- Tombol shortcut default: `A`
 - Caption: `Semuanya menjadi lebih sulit...`
-- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Mode-mode game yang kompatibel: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
 
 ## Deskripsi
 
-Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan aspek-aspek (jika tidak semua) kesulitan dari sebuah [beatmap](/wiki/Beatmap). Mod ini meningkatkan [circle size (CS)](/wiki/Client/Beatmap_editor/Song_Setup#circle-size) sebesar 30%, dan pengaturan kesulitan lainnya sebesar 40%.
+Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan tingkat kesulitan pada sebagian besar aspek dari sebuah [beatmap](/wiki/Beatmap). Modifier ini meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, [approach rate (AR)](/wiki/Beatmapping/Approach_rate) sebesar 40%, [health drain (HP)](/wiki/Beatmapping/Health_drain) sebesar 40% dan [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty) sebesar 40%, meskipun detailnya bervariasi untuk setiap mode permainan.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:30]:
```suggestion
**Hard Rock** merupakan [mod](/wiki/Game_modifier) yang berfungsi untuk meningkatkan tingkat kesulitan suatu [beatmap](/wiki/Beatmap). Mod ini akan memperkecil ukuran [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) senilai 30% serta meningkatkan [approach rate (AR)](/wiki/Beatmapping/Approach_rate), [health drain (HP)](/wiki/Beatmapping/Health_drain), dan [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty) yang aktif senilai 40%, dengan aplikasi yang berbeda-beda untuk masing-masing mode permainannya.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 2]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+
+Untuk menghitung timing window setelah HR diterapkan, [timing window awal](/wiki/Beatmapping/Overall_difficulty) dibagi dengan 1.4. Timing window Rainbow 300 (MAX) tetap tanpa mempedulikan OD, sehingga jendela waktu MAX dengan HR akan selalu ±11,43ms.
+
+Perlu diketahui bahwa metode penghitungannya berbeda pada [ScoreV2](/wiki/Game_modifier/ScoreV2).
 
 ## Trivia
 
-- Mod Hard Rock dinamai dari kesulitan tersusah pada permainan [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS), "[Elite Beat Agents](/wiki/iNiS_games)"
-- Mod Hard Rock akan mematikan mod [Easy](/wiki/Game_modifier/Easy). Dan sebaliknya.
+- Mod Hard Rock dinamai berdasarkan tingkat kesulitan tersulit dari game [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS), "[Elite Beat Agents](/wiki/iNiS_games)"
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:68]:
```suggestion
- Nama mod Hard Rock diambil dari nama tingkat kesulitan tertinggi yang ada permainan "[Elite Beat Agents](/wiki/iNiS_games)" yang dirilis pada konsol [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS), 
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 3]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+- Mod Hard Rock akan mematikan mod [Easy](/wiki/Game_modifier/Easy) dan sebaliknya.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:69]:
```suggestion
- Penggunaan mod Hard Rock akan menonaktifkan mod [Easy](/wiki/Game_modifier/Easy) dan sebaliknya.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 4]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
-![Tampilan permainan HR](img/HR-comparison-catch.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama dengan mod Hard Rock dipasang (kanan) pada mode osu!catch")
+![Tampilan permainan HR catch](img/HR-comparison-catch.jpg "Perbandingan permainan biasa (kiri) dengan permainan dengan mod Hard Rock diaktifkan (kanan) di osu!catch")
 
 ### osu!mania
 
-*Catatan: Menggunakan mod ini pada mode osu!mania akan menyebabkan sebuah unranked play.*
+*Catatan: Menggunakan mod ini pada osu!mania akan menghasilkan unranked play.*
 
-Pada [osu!mania](/wiki/Game_mode/osu!mania), hanya [judgement](/wiki/Gameplay/Judgement) overall difficulty (OD) dan nilai AR yang ditingkatkan. Pengaturan kesulitan lainnya tetap tidak berubah. Posisi peletakkan note-note tidak terpengaruhi.
+PDalam [osu!mania](/wiki/Game_mode/osu!mania), HP drain meningkat sebanyak 40% sementara timing window menjadi lebih ketat sebanyak 29%. Meskipun nilai AR ditampilkan meningkat, hal itu tidak mempengaruhi permainan.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:60]:
```suggestion
Pada mode [osu!mania](/wiki/Game_mode/osu!mania), Hard Rock akan meningkatkan nilai HP drain sebesar 40% dan menjadikan *timing window* yang ada 29% lebih ketat. Meskipun nilai AR juga akan ditampilkan meningkat, hal ini tidak akan mempengaruhi permainan.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 5]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+
+Untuk menghitung timing window setelah HR diterapkan, [timing window awal](/wiki/Beatmapping/Overall_difficulty) dibagi dengan 1.4. Timing window Rainbow 300 (MAX) tetap tanpa mempedulikan OD, sehingga jendela waktu MAX dengan HR akan selalu ±11,43ms.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:62]:
```suggestion
*Timing window* baru setelah HR diterapkan akan sama dengan [*timing window* awal](/wiki/Beatmapping/Overall_difficulty) dibagi dengan 1.4. Meskipun demikian, *timing window* untuk Rainbow 300 (MAX) akan selalu sama terlepas dari OD yang aktif, sehingga *timing window* MAX baik dengan ataupun tanpa HR akan selalu berada pada kisaran ±11,43ms.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 6]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+Mod **Hard Rock** adalah sebuah [game modifier](/wiki/Game_modifier) yang bertujuan untuk meningkatkan tingkat kesulitan pada sebagian besar aspek dari sebuah [beatmap](/wiki/Beatmap). Modifier ini meningkatkan [circle size (CS)](/wiki/Beatmap_Editor/Song_Setup#circle-size) sebesar 30%, [approach rate (AR)](/wiki/Beatmapping/Approach_rate) sebesar 40%, [health drain (HP)](/wiki/Beatmapping/Health_drain) sebesar 40% dan [overall difficulty (OD)](/wiki/Beatmap_Editor/Song_Setup#overall-difficulty) sebesar 40%, meskipun detailnya bervariasi untuk setiap mode permainan.
 
 ### osu!
 
-Pada [osu!](/wiki/Game_mode/osu!), (bersama dengan pengaturan kesulitan yang telah ditingkatkan) mod Hard Rock membalikkan posisi sumbu-X sebuah beatmap (ditunjukkan dibawah). Hal ini kadang-kadang bisa menyebabkan [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak benar, yang bisa meningkatkan kesulitan.
+Pada [osu!](/wiki/Game_mode/osu!), (bersamaan dengan peningkatan pengaturan kesulitan) mod Hard Rock membalikkan beatmap pada sumbu X-nya (seperti yang ditunjukkan di bawah).Hal ini terkadang dapat membuat [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak alami, yang dapat meningkatkan kesulitan.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:34]:
```suggestion
Pada mode [osu!](/wiki/Game_mode/osu!), di samping membuat suatu beatmap menjadi lebih sulit, mod Hard Rock juga akan mencerminkan seluruh objek yang ada beatmap terhadap sumbu-X (sebagaimana yang dicontohkan di bawah ini). Hal ini terkadang dapat membuat [pola](/wiki/Beatmap/Pattern) tertentu terasa tidak alami dan lebih sulit untuk dimainkan.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 7]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
 
-Sebagai tambahan, jika pemain telah menghafal atau sudah terbiasa dengan gerakan dan pola sebuah beatmap, beralih ke mod Hard Rock bisa menghilangkan penghafalan tersebut dan menyebabkan semacam sebuah kesulitan palsu.
+Selain itu, jika pemain telah menghafal atau terbiasa dengan gerakan dan pola beatmap, beralih ke mod Hard Rock dapat menghilangkan hafalan tersebut dan menyebabkan semacam kesulitan buatan.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:36]:
```suggestion
Di samping itu, apabila seorang pemain telah menghafal atau membiasakan diri dengan gerakan dan pola yang ada pada suatu beatmap, beralih ke mod Hard Rock dapat menghilangkan hafalan tersebut dan menimbulkan kesulitannya tersendiri.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 8]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
 
-![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama dengan mod Hard Rock dipasang (kanan) pada mode osu!")
+![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan permainan biasa (kiri) dengan permainan dengan mod Hard Rock diaktifkan (kanan) pada mode osu!")
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:38]:
```suggestion
![Tampilan permainan HR](img/HR-comparison-osu.jpg "Perbandingan antara permainan biasa (kiri) dan permainan dengan mod Hard Rock yang aktif (kanan) pada mode osu!")
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 9]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
 
 ### osu!taiko
 
-Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock meningkatkan [slider velocity](/wiki/Hit_object/Slider_velocity) dan juga [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty). Mod ini adalah mod yang biasa digunakan secara luas pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan penghafalan atau mengetuk dengan sangat cepat pada tingkat kesulitan yang lebih tinggi.
+Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock meningkatkan [slider velocity](/wiki/Hit_object/Slider_velocity) serta [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty).Mod ini dianggap secara luas sebagai mod yang paling umum digunakan pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan penghafalan atau mengetuk dengan sangat cepat pada tingkat kesulitan yang lebih tinggi.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:42]:
```suggestion
Pada [osu!taiko](/wiki/Game_mode/osu!taiko), mod Hard Rock akan meningkatkan *[slider velocity](/wiki/Hit_object/Slider_velocity) dasar serta [overall difficulty (OD)](/wiki/Beatmapping/Overall_difficulty) yang aktif. Secara umum, mod ini merupakan mod yang paling sering digunakan pada osu!taiko karena mod ini adalah satu-satunya mod peningkat kesulitan yang tidak membutuhkan pemain untuk menghafal ataupun mengenai objek yang ada dengan sangat cepat.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 10]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
+Namun, efek ini seringnya tidak signifikan karena setiap hit dalam denden memberikan skor statis 300. Yang mana lebih kecil dari perbedaan skor biasa antara hit `GREAT` dan `GOOD` (550 poin pada 100x kombo atau lebih tinggi).
 
 ### osu!catch
 
-Pada osu!catch, mod Hard Rock memiliki beberapa efek. Selain dari memperkecil ukuran pemain dan fruit dan meningkatkan kecepatan jatuh mereka, mod ini bisa mengakibatkan fruit biasa bergeser sedikit ke kanan atau kiri dari posisi awal mereka pada layar. Perubahan ini menyebabkan pemain untuk bergerak lebih banyak dan lebih sering menggunakan hyperdash, yang mana juga meningkatkan kesulitan. 
+Pada osu!catch, mod Hard Rock memiliki beberapa efek. Selain membuat karakter pemain dan buah-buahan menjadi lebih kecil dan meningkatkan kecepatan jatuhnya, mod ini bisa mengakibatkan fruit biasa bergeser sedikit ke kiri atau kanan dari posisi awal mereka di layar. Perubahan ini mengharuskan pemain untuk bergerak lebih banyak dan lebih sering menggunakan hyperdash,yang juga dapat meningkatkan kesulitan.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:50]:
```suggestion
Pada mode osu!catch, mod Hard Rock akan berpengaruh terhadap beberapa hal. Di samping membuat *catcher* dan buah yang ada menjadi lebih kecil dan lebih cepat jatuh, mod ini dapat mengakibatkan buah-buah tertentu untuk tergeser sedikit ke kiri atau kanan dari posisi mereka seharusnya. Perubahan ini seringkali menuntut pemain untuk lebih aktif bergerak dan menggunakan *hyperdash*, yang secara tidak langsung juga akan meningkatkan kesulitan yang ada.
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 11]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
 
-Dikarenakan kenaikan pada tingkat kesulitan yang lebih besar dibandingkan dengan [mode-mode game](/wiki/Game_mode) yang lainnya, mod Hard Rock bernilai 1.12x pengali skor daripada 1.12x pengali skor pada mode-mode permainan yang lainnya.
+Dikarenakan kenaikan tingkat kesulitan yang lebih besar dibandingkan dengan [mode permainan](/wiki/Game_mode) lainnya, mod Hard Rock bernilai 1.12x pengali skor dibandingan dengan 1.06x pengali skor pada semua mode permainan lainnya.
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:52]:
```suggestion
Karena mod Hard Rock pada mode osu!catch jauh lebih memengaruhi tingkat kesulitan suatu beatmap dibanding mod Hard Rock pada [mode permainan](/wiki/Game_mode) lainnya, mod Hard Rock pada mode osu!catch memiliki pengali skor yang lebih besar (1.12x) dibanding dengan mode permainan lainnya (1.06x).
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 12]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
...
 
-![Tampilan permainan HR](img/HR-comparison-catch.jpg "Perbandingan dari sebuah map tanpa mod terpasang (kiri) dengan map yang sama dengan mod Hard Rock dipasang (kanan) pada mode osu!catch")
+![Tampilan permainan HR catch](img/HR-comparison-catch.jpg "Perbandingan permainan biasa (kiri) dengan permainan dengan mod Hard Rock diaktifkan (kanan) di osu!catch")
```
Comment (Nivalyx) [inline wiki/Gameplay/Game_modifier/Hard_Rock/id.md:54]:
```suggestion
![Tampilan permainan HR catch](img/HR-comparison-catch.jpg "Perbandingan antara permainan biasa (kiri) dan permainan dengan mod Hard Rock yang aktif (kanan) pada mode osu!catch")
```

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 13]
```diff
@@ -5,15 +5,13 @@ tags:
...
-outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
 ---
 
 # Hard Rock (mod)
 
-![Mod ikon HR](/wiki/shared/mods/HR.png "Mod ikon Hard Rock (HR)")
+![Mod ikon HR](/wiki/shared/mods/HR.png "Ikon mod Hard Rock (HR)")
 
-*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/Game_modifier)*
+*Untuk melihat daftar keseluruhan dari semua mod, lihat: [Game Modifier](/wiki/Game_modifier)*
```
[the remark this answers is not in the document]
Comment (daud-ahn) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:14]: Wikipedia uses 'lihat' for things like this. From what I see, 'lihat' and 'kunjungi' are both used on Indonesian wiki pages. I think they should be standardised.

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md [hunk 14]
```diff
@@ -23,48 +21,52 @@ outdated_since: bb68816d3f5cdcce9eaa78788e9fc7b671e1dc93
   - ![][osu!] ![][osu!taiko]: 1.06x
   - ![][osu!catch]: 1.12x
   - ![][osu!mania]: 1.00x
-- Tombol Shortcut Default: `A`
+- Tombol shortcut default: `A`
 - Caption: `Semuanya menjadi lebih sulit...`
-- Mode-mode game yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+- Mode-mode game yang kompatibel: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
```
[the remark this answers is not in the document]
Comment (daud-ahn) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:26]: 'kompatibel' is listed in KBBI. I think it's okay to use it.

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md (hunk 9 above)
Comment (daud-ahn) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:42]: slider velocity is italicised but not overall difficulty?

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (Nivalyx) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:26]: hmm it's your call i think, both "sesuai" and "kompatibel" should be feasible here

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md (hunk 9 above)
Comment (Nivalyx) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md]: hmm you can un-italicize both if you're looking for consistency i think

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (Nivalyx) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:14]: yeah, this is similar to http://github.com/ppy/osu-wiki/pull/7514#discussion_r899023802 -- both words are interchangeable (e.g. none of them are wrong) and people have basically  been using whichever they think is more suitable in regards to the situation.

it's up to you whether to use "lihat" or "kunjungi" tho

File: wiki/Gameplay/Game_modifier/Hard_Rock/id.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (daud-ahn) [reply wiki/Gameplay/Game_modifier/Hard_Rock/id.md:26]: I thought I could get a definitive answer in lazer, but the Indonesian translation is incomplete.

**Review by Nivalyx — APPROVED** [review]
should be good i guess

**Review by ZeroPyrozen — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**